spanner/ddl/ast.go

```diff
@@ -537,8 +537,8 @@ func isStoredColumnKeyPartOfPrimaryKey(ct CreateTable, colId string) bool {
 func (k Foreignkey) PrintForeignKeyAlterTable(spannerSchema Schema, c Config, tableId string) string {
 	var cols, referCols []string
 	for i, col := range k.ColIds {
```

Comment:
Please add quotes to [check constraints](https://github.com/GoogleCloudPlatform/spanner-migration-tool/blob/525dd1ab35c1bb55cd8d73cc87ea3d82a640fdf0/spanner/ddl/ast.go#L560) as well

Reply:
Check constraints already adds the backticks.